dashboard.py

```diff
@@ -18,48 +18,83 @@ class MainApplication(tk.Frame):
 
     """
 
-    def __init__(self, parent, gateway, sensors, *args, **kwargs):
+    def __init__(self, parent, gateway, *args, **kwargs):
         tk.Frame.__init__(self, parent, *args, **kwargs)
         self.parent = parent
         self.gateway = gateway
-        self.sensors = sensors
+        self.sensors = self.gateway.sensors
 
-        self.gateway_messages = MessageBox(
-            self, gateway=self.gateway, borderwidth=2, relief="groove")
-        self.graph = LiveTimeGraph(
-            self, gateway=self.gateway, sensor=self.sensors.imu, field="velocity")
-        self.gateway_status = GatewayStatus(self, gateway=self.gateway)
+        self.speed_graph = LiveTimeGraph(
+            self, self.gateway, self.sensors.bmp2, field="Temperature")
```

Comment:
Speed_graph or Temperature? I think it should be field="Velocity" or ="Speed"

Reply:
Yes! Will be updated once we have the real plots